src/components/Reports/UploadPhotos.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Container } from '@mui/material';
+import SelectPhotos from 'src/components/common/SelectPhotos';
+
+type UploadPhotosProps = {
+  onPhotosChanged: (photos: File[]) => void;
+};
+export default function UploadPhotos(props: UploadPhotosProps): JSX.Element {
+  const { onPhotosChanged } = props;
+
+  return (
+    <Container maxWidth={false}>
+      <SelectPhotos onPhotosChanged={onPhotosChanged} multipleSelection={true} />
+    </Container>
```

Comment:
this is fine as a component but maybe can just not be a separate component? only addition here is the container

Reply:
Yes, I think I created this because I thought it was going to do a lot of stuff and then it resulted really simple